### PR TITLE
WIP Refactor network

### PR DESCRIPTION
**Pull request by msimonin**

### CODE REVIEW

**Comment by rcherrueau** [comment]
I cannot ping VMs on their floating ip.

File: docs/provider/static.rst [hunk 1]
```diff
@@ -25,22 +25,21 @@ already running resources.
...
-       end:     # Last available ip
-       cidr:    # cidr notation to describe the pool of ips
-       gateway: # Ip address of the gateway
-       dns:     # Ip address of the DNS
-     eths:
-       - eth1   # Name of the APIs interface
-       - eth2   # Name of the external interface
+     networks: # An array of networks
+     # one network looks like the following
+       cidr: 192.168.0.0/24,
```
Comment (rcherrueau) [inline docs/provider/static.rst]: Do you omit to add a dash `- ` in front of `cidr` to define an array, or is it normal?

Comment (msimonin) [inline docs/provider/static.rst]: It's still WiP :)

**Comment by msimonin** [comment]
updated the G5K provider.

**Comment by msimonin** [comment]
closed in favour of beyondtheclouds/enoslib

**Pull request closed without merging**